.github/linters/ct.yaml

```diff
@@ -12,4 +12,5 @@ excluded-charts:
   # If not running on GCE, will error: "Failed to get GCE config"
   - prometheus-to-sd
 additional-commands:
+  - sh -c "if [ -f '{{ .Path }}/ci/lint.sh' ]; then exec bash '{{ .Path }}/ci/lint.sh'; fi"
```

Comment:
@jkroepke @monotek What do you think of running shellcheck on `ci/lint.sh` before executing it? It's already included on [GH Runners image](https://github.com/actions/runner-images/blob/83c87d632c09ac7ea61b150e9df8f794dde85391/images/ubuntu/Ubuntu2204-Readme.md?plain=1#L369).

Reply:
Make sense to me!

Reply:
Created https://github.com/prometheus-community/helm-charts/pull/4072